### ISSUE TITLE
regression in build tests - MNIST notebook

### ISSUE THREAD

**Issue by mvesin** [fedbiomed/fedbiomed#935]
A regression was introduced in automatic build test by removing `notebooks/101_getting_started.py`.

This script is used by automatic build tests via `scripts/run_test_mnist` each time we do a merge. This test is now empty and issuing in the logs:

```
INFO: no script /home/ci/actions-runner/_work/fedbiomed/fedbiomed/notebooks/101_getting-started.py in this version, skipping
```

Expected fix:
- re-introduce 101_getting-started.py` or create it automatically via `jupyter nbconvert` in script `run_test_mnist`
- review & fix all other occurrences referencing `101_getting-started.py` in documentations (in README.md, docs, env/README.md, tests/README.md)

**Comment by ErwanDemairy** [comment]
Update to add to the documentation alongside:
1. [ ] remove the cells containing, in order to allow python running of the scipts:
```ipython
Use for developing (autoreloads changes made across packages)
#%%
%load_ext autoreload
%autoreload 2
```
2. [ ] add in the readme the snippet or in the developer documentation.

**Comment by ErwanDemairy** [comment]
Should the notebooks in the documentation directories also be runnable by python, once nb-converted?
Currently, there are 3 notebooks including the autoreload snippet.
```bash
$ grep -r autorel *                  
docs/tutorials/security/non-private-local-central-dp-monai-2d-image-registration.ipynb:    "%load_ext autoreload\n",
docs/tutorials/security/non-private-local-central-dp-monai-2d-image-registration.ipynb:    "%autoreload 2"
docs/tutorials/monai/02_monai-2d-image-registration.ipynb:    "%load_ext autoreload\n",
docs/tutorials/monai/02_monai-2d-image-registration.ipynb:    "%autoreload 2"
docs/tutorials/scikit-learn/02_sklearn_sgd_regressor_tutorial.ipynb:    "%load_ext autoreload\n",
docs/tutorials/scikit-learn/02_sklearn_sgd_regressor_tutorial.ipynb:    "%autoreload 2"
```

**Comment by ErwanDemairy** [comment]
Solved by #937
